README.md

```diff
@@ -35,16 +35,80 @@ gradle bootRun
 This starts the embedded Tomcat server on port 8080.
 
 
-~~**NOTE**: A single user with username 'samwise' and password 'gamgee' has been configured in the _application.properties_ file.~~ **Security temporarily disabled**
```

Comment:
Adding: `spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults=false` into `application.properties` will prevent the following from appearing at startup (as you know, it's an ignorable error but it might help to avoid some confusion):

![exception](https://user-images.githubusercontent.com/10905039/39965098-cbfa4fe4-5647-11e8-8f23-cabd26947495.png)